test/test_helpers/data_migrations.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+# Test your data migrations as follows.
+#
+# require 'test_helper'
+#
+# module DataMigrations
+#   class MyDataMigrationTest < DataMigrationTest
+#     test 'does something' do
+#       migration = MyDataMigration.new
+#       assert_something do
+#         migration.up
+#       end
+#     end
+#   end
+# end
+
+module DataMigrations
+  class DataMigrationTest < ActiveSupport::TestCase ; end
+end
+
+RailsDataMigrations::LogEntry.create_table unless ActiveRecord::Base.connection.table_exists? RailsDataMigrations::LogEntry.table_name
```

Comment:
Is this table the same as our log_entries? It might be confusing or lead to errors

Reply:
No, it's a different one, named `data_migrations`.